packages/panzer/disc-fe/test/evaluator_tests/integrator_scalar.cpp

```diff
@@ -105,16 +107,17 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL(integrator_scalar_side,test2d,EvalType)
   MDFieldArrayFactory af("",true);
   workset->cell_vertex_coordinates = af.buildStaticArray<double,Cell,NODE,Dim>("coords",numCells,numVerts,dim);
   Workset::CellCoordArray coords = workset->cell_vertex_coordinates;
+  auto coords_v = coords.get_static_view();
   Kokkos::parallel_for(1, KOKKOS_LAMBDA (int) {
```

Comment:
Did that get rid of a warning?  Or just more clear?

Reply:
It got rid of the warnings. lambda capture of MDField generates a warning since it has an RCP member internally. Pulling out the raw view cleans it up.

Reply:
note that we don't see the warnings for mdfields with kokkos functors due to how the functor is moved to device. Just see the warnings for lambdas.